course_discovery/apps/api/v1/views/search.py

```diff
@@ -197,9 +197,10 @@ class PersonSearchViewSet(BaseHaystackViewSet):
     """
     Generic person search
     """
+    ordering_fields = ('created', 'full_name',)
```

Comment:
Similar to course title ordering we have to make sure the `full_name` of instructors follow the same pattern, e.g camel case

Reply:
Yes, we’ve the same issue here as well but here we can modify our WordPress loader to use Python title regardless of whatever case our user used in WordPress while creation instructor. Should I add the change for that in the current PR?

Reply:
From slack conversation, we would need to add the change in current PR.
https://edlyinc.slack.com/archives/C010MM9C282/p1621850534032900?thread_ts=1621845483.025900&cid=C010MM9C282

Reply:
Updated.